devel/ldid-procursus/Portfile

```diff
@@ -0,0 +1,34 @@
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                ldid-procursus
+version             2.1.5-procursus7
+categories          devel
+license             AGPL-3
+platforms           darwin
+conflicts           ldid
+maintainers         @therealketo procurs.us:team
+description         Put real or fake signatures in a Mach-O.
+long_description    ${description}
+homepage            https://github.com/ProcursusTeam/ldid
+
+github.setup        ProcursusTeam ldid ${version} v
+github.tarball_from archive
+checksums           rmd160 648697e84ac2b71fff24fdc1241f90d7390124d5 \
+                    sha256 04e461c6f02765e48fc9cc0b68d4dc353a9c46bc1c4d8bac0695509d1af1ff5e \
+                    size 53532
+
+depends_build       port:pkgconfig
+depends_lib         path:lib/libplist-2.0.dylib:libplist \
+                    path:lib/libcrypto.3.dylib:openssl
+
+use_configure       no
+universal_variant   no
+use_xcode           no
+destroot.args       PREFIX=${prefix}
+
+post-destroot {
+    set zsh_comp_path ${destroot}${prefix}/share/zsh/site-functions
+    xinstall -d ${zsh_comp_path}
+    xinstall -m 0644 ${worksrcpath}/_ldid ${zsh_comp_path}
```

Comment:
Some ports uses `zsh_completion` variant

Reply:
Should the variant depend on zsh? I see some Portfiles append it as a runtime dependency.